crates/re_types_builder/src/objects.rs

```diff
@@ -68,28 +68,34 @@ impl Objects {
             objects: resolved_enums.into_iter().chain(resolved_objs).collect(),
         };
 
-        // Validate fields types: Archetype consist of components, everything else consists of datatypes.
+        // Validate fields types: Archetype consist of components, Space Views consist of archetypes, everything else consists of datatypes.
```

Comment:
I can't help but wonder if "SuperArchetypes" (structured collections of archetype) are a generic things of which SpaceViews just happen to be the first example.